tests/test_writer.py

```diff
@@ -92,10 +93,16 @@ def test_iter_lines__unsafe_dependencies(from_line, writer):
                                reverse_dependencies,
                                ['test'],
                                {},
-                               None)
-    str_lines = []
-    for line in lines:
-        str_lines.append(line)
+                               None,
+                               allow_unsafe=allow_unsafe)
+    str_lines = list(lines)
     assert comment('# The following packages are considered to be unsafe in a requirements file:') in str_lines
-    assert comment('# setuptools') in str_lines
+    if allow_unsafe:
+        assert any('#    pip-compile --allow-unsafe --output-file ' in str_line
+                   for str_line in str_lines)
```

Comment:
I would prefer:
```
assert comment('#    pip-compile --output-file dst_file src_file src_file2') in str_lines
```

Reply:
Sounds reasonable to me. I had been trying not to assert much beyond the part of the output that changes, but that's not so bad.